### ISSUE TITLE
CG mode mistakenly sees subreadings as ambiguous because of extra spaces

### ISSUE THREAD

**Issue by ftyers** [jonorthwash/ud-annotatrix#228]
If you make sentence that is not ambiguous, but has subreadings, the CG mode detects it as ambiguous. This is because converting from CONLLU to CG adds spaces before tags.

![captura de 2018-01-14 14-41-09](https://user-images.githubusercontent.com/449545/34916459-06865502-f939-11e7-9ca5-98675caa55a3.png)

**Comment by jonorthwash** [comment]
Could you share example text that triggers this?

**Comment by ftyers** [comment]
@jonorthwash It's in the picture :D The classic "I wanna skate" example.

**Comment by jonorthwash** [comment]
I mean, could you provide the exact text you're using?

**Comment by ftyers** [comment]
```
# sent_id = _
# text = I wanna skate to the beach .
"<I>"
	 PRON @x #1->2
"<wanna>"
	 VERB @root #2->0
		 PART @x #3->4
"<skate>"
	 VERB @x #4->2
"<to>"
	 ADP @x #5->7
"<the>"
	 DET @det #6->7
"<beach>"
	 NOUN @x #7->4
"<.>"
	 PUNCT @punct #8->2
```

**Comment by jonorthwash** [comment]
Does it work okay if you get rid of the extra spaces?
```
# sent_id = _
# text = I wanna skate to the beach .
"<I>"
	PRON @x #1->2
"<wanna>"
	VERB @root #2->0
		PART @x #3->4
"<skate>"
	VERB @x #4->2
"<to>"
	ADP @x #5->7
"<the>"
	DET @det #6->7
"<beach>"
	NOUN @x #7->4
"<.>"
	PUNCT @punct #8->2
```
Seems to for me:
![2018-01-17-14 25 05_001](https://user-images.githubusercontent.com/963849/35062413-3f492198-fb92-11e7-9646-655d62645053.png)

**Comment by jonorthwash** [comment]
I'm not sure why I get all that extra space above the graph though.

**Comment by ftyers** [comment]
try switching to conllu then switching back to CG

**Comment by jonorthwash** [comment]
Aha.  It seems to be adding a space.  So the fact that it's adding a space to all parses is the bug, not that it's misinterpreting subreadings....

**Comment by jonorthwash** [comment]
Okay, now that the entire thread has been changed, this bug is filed correctly....

**Comment by ftyers** [comment]
It's because there are no lemmas I think...

**Comment by ftyers** [comment]
```
function newCgAna(i, tok) {
    var lemma = (tok.lemma) ? ('"' + tok.lemma + '"') : '';

...
    var cgToken = lemma + " " + pos + feats + deprel + " #" + tok.id + "->" + head;

```

**Comment by ftyers** [comment]
Fixed in a1db094
